src/sections/Member-single/index.js

```diff
@@ -23,24 +23,24 @@ const MemberSingle = ({frontmatter}) =>{
                                 { position && (
                                     <h2>{position}</h2>
                                 )}
+                            </header>
+                            <div className="body">
                                 {
                                     location &&
                                     <h4>Location: {location}</h4>
                                 }
 
                                 {
                                     bio &&
-                                    <h6>{bio}</h6>
+                                    <h4>{bio}</h4>
```

Comment:
Can you avoid changing the font-weight of the header tags (also do other headers tags)? You can create a class and use that in a `div` or a `span`

Reply:
This is left